From ae84cd3cc3ed258470d0f8c3b891181a93594620 Mon Sep 17 00:00:00 2001
From: John Greene <jogreene@redhat.com>
Date: Thu, 14 Aug 2014 14:42:13 -0400
Subject: [ethernet] e1000e: fix message terminations

Message-id: <1408027348-9610-20-git-send-email-jogreene@redhat.com>
Patchwork-id: 87710
O-Subject: [RHEL7.1 PATCH 19/34] [e1000e] e1000e/igb/ixgbe/i40e: fix message terminations
Bugzilla: 1091119
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

[redhat: e1000e parts only]
commit c5ffe7e1f745984b37b8ffe03b03f3d716a072f3

    Add \n at the end of messages where missing, remove all \r.

Reported-by: Joe Perches <joe@perches.com>
Signed-off-by: Jakub Kicinski <kubakici@wp.pl>
Tested-by: Aaron Brown <aaron.f.brown@intel.com>
Tested-by: Phil Schmitt <phillip.j.schmitt@intel.com>
Tested-by: Kavindya Deegala <kavindya.s.deegala@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>

BZ: 1091119
Testing: Acceptable, see BZ
Brew: https://brewweb.devel.redhat.com/taskinfo?taskID=7834849
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/e1000e/netdev.c b/drivers/net/ethernet/intel/e1000e/netdev.c
index 13e1baf..9eae8b2 100644
--- a/drivers/net/ethernet/intel/e1000e/netdev.c
+++ b/drivers/net/ethernet/intel/e1000e/netdev.c
@@ -1168,7 +1168,7 @@ static void e1000e_tx_hwtstamp_work(struct work_struct *work)
   dev_kfree_skb_any(adapter->tx_hwtstamp_skb);
   adapter->tx_hwtstamp_skb = NULL;
   adapter->tx_hwtstamp_timeouts++;
-  e_warn("clearing Tx timestamp hang");
+  e_warn("clearing Tx timestamp hang\n");
  } else {
   /* reschedule to check later */
   schedule_work(&adapter->tx_hwtstamp_work);
-- 
1.7.1